docs/conf.py

```diff
@@ -66,9 +66,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'POC-0'
+version = u'Developer Preview'
```

Comment:
We still need to decide on the version number. Developer preview is just our nickname for it. The actual version number should be following [semantic versioning](http://semver.org/).

I would do it `0.1.0`